charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+  {{- if .Values.ui.labels }}
+{{ toYaml .Values.ui.labels | indent 4}}
+  {{- end}}
+  name: {{ template "cp-schema-registry.fullname" . }}-ui
+spec:
+  replicas: {{ .Values.ui.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "cp-schema-registry.fullname" . }}
+      service: schema-registry-ui
+  template:
+    metadata:
+      {{- if .Values.ui.annotations }}
+      annotations:
+{{ toYaml .Values.ui.annotations | indent 8 }}
+      {{ end }}
+      labels:
+        app: {{ template "cp-schema-registry.fullname" . }}
+        service: schema-registry-ui
+      {{- if .Values.ui.labels }}
+{{ toYaml .Values.ui.labels | indent 8}}
+      {{- end}}
+    spec:
+      securityContext:
+      {{- if .Values.securityContext }}
+{{ toYaml .Values.securityContext | indent 8 }}
+      {{- end}}
+      containers:
+        - env:
+            - name: PROXY
+              value: "true"
+            - name: SCHEMAREGISTRY_URL
+              value: {{ template  "cp-schema-registry.connectUrlValue" . }}
+          image: landoop/schema-registry-ui
+          imagePullPolicy: IfNotPresent
+          name: ui
+          ports:
+            - containerPort: 8000
+              protocol: TCP
+          resources:
+{{ toYaml .Values.ui.resources | indent 12 }}
```

Comment:
Here, using `nindent` would be more readable (because not breaking the indentation).
It would look like this:
```
          ports:
            - containerPort: 8000
              protocol: TCP
          resources: {{ toYaml .Values.ui.resources | nindent 12 }}
```